README.md

```diff
@@ -4,7 +4,15 @@
 
 1. Submit a new [access request issue](https://github.com/rust-vmm/community/issues/new?assignees=andreeaflorescu%2C+jbryce%2C+sameo&labels=&template=access-request-to-rust-vmm-organization.md&title=Access+Request+to+rust-vmm).
 
-## Adding a new repository
+## How to contribute?
+
+### Adding a new crate to rust-vmm
+
+Each rust-vmm crate lives in its own repo under the [rust-vmm organization](http://github.com/rust-vmm).
+If you want to contribute to rust-vmm by adding a new Rust crate to the project, you will have to go through the following steps:
+
+1. Submit a [crate addition request](https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=new-crate-request.md&title=Crate+Addition).
+2. The crate relevance to the project will be discussed through the issue submitted in step 1.
+3. If the crate gets accepted by the project maintainers, a new, empty repository will be created under the rust-vmm repository.
+4. Submit a pull request to the newly created repository as your initial code submission.
```

Comment:
Just a comment here, please make sure we don't receive a huge PR as the first PR, otherwise it'll be impossible to review.
Maybe we should specify the code should be properly split, and don't expect the crate to be fully functional after the first PR has been merged.

Reply:
I agree, but I think this is a generic statement and should apply to **all** rust-vmm PRs. And this should be described e.g. in a `CONTRIBUTING` document in this repo.

Reply:
:+1: